crypto/threads_none.c

```diff
@@ -121,4 +121,10 @@ int CRYPTO_atomic_add(int *val, int amount, int *ret, CRYPTO_RWLOCK *lock)
     return 1;
 }
 
+int openssl_init_fork_handlers(void)
+{
+    /* Is failing the right thing to do? */
```

Comment:
If the default is to fail, then users of the none-type threading must add OPENSSL_INIT_NO_ATFORK.